src/widgets/WebexMeeting/README.md

```diff
@@ -16,10 +16,21 @@ To get a preview of the Webex Meeting widget, you can run our start script:
 
 ## Embed
 
-Import the Meeting widget from the `@webex/widgets` library and embed it in your React application
+Import the Meeting widget from the `@webex/widgets` library and embed it in your React application.
+`WebexMeetingWidget` takes an access token as a string, a meeting destination as a string and an optional function to define custom controls for a meeting. The `controls` function should expect a boolean parameter that indicates whether the meeting is active and returns an array of control names to display on active (in-meeting) or inactive (interstitial) state of the meeting.The default control names are set to `['mute-audio', 'mute-video', 'settings', 'join-meeting']` if the meeting is inactive and `['mute-audio', 'mute-video', 'share-screen', 'member-roster', 'settings', 'leave-meeting']`otherwise.
```

Comment:
Let's move the control customization into a different section. This is a big block of text and it's hard to digest.
So, let's leave this as-is
![image](https://user-images.githubusercontent.com/6610308/135944440-1fcda5a3-de81-49a9-8b58-1599a53f9281.png)
And create a new sub-section:
```suggestion
### Customize Meeting Controls

`WebexMeetingWidget` takes an optional function to define custom controls for a meeting. The `controls` function should expect a boolean parameter that indicates whether the meeting is active and returns an array of control names to display on active (in-meeting) or inactive (interstitial) state of the meeting.
The default control names are set to `['mute-audio', 'mute-video', 'share-screen', 'member-roster', 'settings', 'leave-meeting']` if meeting is active and `['mute-audio', 'mute-video', 'settings', 'join-meeting']` otherwise.
This meeting controls come from the [SDK adapter](https://github.com/webex/sdk-component-adapter). 

Use `supportedControls()` of the meetings adapter to obtain the names of the implemented controls by the adapter.

To maintain the same controls regardless of meeting state, ignore the boolean parameter passed to your control function.

```js
const myControls = (isActive) => isActive ? ['leave-meeting'] : ['join-meeting'];

<WebexMeetingWidget 
  acessToken="<YOUR_ACCESS_TOKEN>" 
  meetingDestination="meetingDestination" 
  controls={myControls} 
/>
\```
```